src/experimental_features.cpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2020 The Zcash developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or https://www.opensource.org/licenses/mit-license.php .
+
+#include "experimental_features.h"
+
+#include "util.h"
+
+bool fExperimentalMode = false;
+bool fDeveloperEncryptWallet = false;
+bool fDeveloperSetPoolSizeZero = false;
+bool fPaymentDisclosure = false;
+bool fInsightExplorer = false;
```

Comment:
`fExperimentalMode` is no longer used anywhere outside this file (by design), so it should be removed (and turned into a local inside `InitExperimentalMode`. The other experimental booleans should be named using the pattern `fExperimentalFooBar` so it's clear at the gating point what the global is doing.

Reply:
I removed `fExperimentalMode` in https://github.com/zcash/zcash/pull/4293/commits/efcc1a809abd99ddbcabb5009c1afbc949db1107 and i found a few places missed to refactor to the new logic.

Reply:
The rename of the flags was done at https://github.com/zcash/zcash/pull/4293/commits/ce2fd0526437df0a11821355e6dc187c69b9db1a